--- keybinder-3.0-0.3.2/docs/keybinder-docs.sgml.orig	2017-02-08 22:54:10.000000000 +0100
+++ keybinder-3.0-0.3.2/docs/keybinder-docs.sgml	2020-05-02 19:46:17.046633851 +0200
@@ -21,10 +21,6 @@
         <xi:include href="xml/keybinder.xml"/>
 
   </chapter>
-  <chapter id="object-tree">
-    <title>Object Hierarchy</title>
-     <xi:include href="xml/tree_index.sgml"/>
-  </chapter>
   <index id="api-index-full">
     <title>API Index</title>
     <xi:include href="xml/api-index-full.xml"><xi:fallback /></xi:include>
